src/services/SessionService.js

```diff
@@ -101,5 +101,26 @@ export default {
 
         throw errorFromHttpResponse(resp);
       });
+  },
+
+  getLatestSession(context, user) {
+    return NetworkService.latestSession(context, { user_id: user._id })
+      .then(resp => {
+        const { sessionId, data } = resp.data || {};
+        const { type, subTopic } = data;
+
+        if (type && subTopic && sessionId) {
+          this.currentSession.sessionId = sessionId;
+          this.currentSession.data = data;
```

Comment:
Is there a need to store this data on the service's `currentSession` prop? It'd be nice to only store it in Vuex

Reply:
Just to add to this change request -- could you add a visual indicator to tell the student why the button is disabled? Don't need a countdown, just a static message

Reply:
@jkeat,

I went with adding a header (similar to the one when there's a current session active) to display a static message. It's just letting the user know how much time they have left before they can request a new session. The amount of minutes gets updated when the StudentCard component is mounted

<img width="1135" alt="Screen Shot 2020-03-31 at 12 24 22 PM" src="https://user-images.githubusercontent.com/35900194/78050689-9b63bb00-734a-11ea-88ee-6a000352fa6a.png">